build/distributions/CentOS/7/x86_64/release_mappings.yaml

```diff
@@ -6,21 +6,21 @@ simp_releases:
     flavors:
       CentOS:
         isos:
-          - name:     'CentOS-7-x86_64-DVD-1611.iso'
-            size:     4379901952
-            checksum: 'c455ee948e872ad2194bdddd39045b83634e8613249182b88f549bb2319d97eb'
-        build_command: 'bundle exec rake build:auto[CentOS-7-x86_64-DVD-1611.iso,6.X]'
+          - name:     'CentOS-7-x86_64-DVD-1708.iso'
```

Comment:
I get a bunch of repo closure errors for OS modules when I do this, but work s with 1611 ISO.  See SIMP-3805.